alpine.dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM alpine:3.16
+LABEL maintainer "Fluentd developers <fluentd@googlegroups.com>"
+LABEL description="Fluentd aggregator OCI image based on Fluentd v1.15.2" version="0.0.1"
```

Comment:
Do you want the base version of Fluentd to be configurable via a build arg?

Note that the various actions can overwrite labels, and indeed set a load of the default OCI ones so it may be worth checking them.

Reply:
> Do you want the base version of Fluentd to be configurable via a build arg?

I don't think we can do this as all of the other package versions need changing in coordination with this (it's also no longer in the description anyway).